src/test/java/api/requests/InstanceRequestsAPICreationTests.java

```diff
@@ -893,4 +921,11 @@ private void placeHoldRequest(IndividualResource item,
       is(ItemStatus.CHECKED_OUT.getValue()));
   }
 
+  private void createTwentyCheckedOutItemCopies(IndividualResource holdings,
+    IndividualResource locationsResource) {
+    IntStream.range(0, 20)
+      .forEach(index -> itemsFixture.basedUponDunkirkWithCustomHoldingAndLocationAndCheckedOut(
+        holdings.getId(), locationsResource.getId()));
+  }
+
```

Comment:
Extract upper range limit into parameters

Reply:
The method is removed.